src/components/TodoList/TodoList.tsx

```diff
@@ -14,87 +26,48 @@ export const TodoList: React.FC = () => (
         <th> </th>
       </tr>
     </thead>
+    {todos.map(todo => (
+      <tbody key={todo.id}>
+        <tr data-cy="todo" className="">
```

Comment:
```suggestion
   <tbody >
    {todos.map(todo => (
        <tr data-cy="todo" key={todo.id}>
```
you need to map the tr tag